hawc/apps/common/backends.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth.backends import BaseBackend
+from rest_framework.authentication import get_authorization_header
+from rest_framework.authtoken.models import Token
+
+from hawc.apps.myuser.models import HAWCUser
+
+
+class TokenBackend(BaseBackend):
```

Comment:
perhaps better to inherit from the `ModelBackend` and just override the authenticate method? That way we can remove the `get_user` method and the other permissions fields will already be implemented